hikey_debian.mk

```diff
@@ -463,7 +463,7 @@ recovery:
 	python $(ROOT)/burn-boot/hisi-idt.py --img1=$(LLOADER_PATH)/l-loader.bin
```

Comment:
I suppose `hikey_debian.mk` needs the same changes as `hikey.mk`.

Reply:
Oops yes. Will fix. Thanks.